src/components/App/App.jsx

```diff
@@ -119,16 +122,17 @@ class App extends React.Component {
             <Route exact path="/search" component={Search} />
             <Route exact path="/records/:info?" component={Records} />
             <Route exact path="/meta" component={Meta} />
+            <Route exact path="/api" component={Api} />
```

Comment:
is there a link to this new route?  might want to move "API Docs" from the footer to the header and link to this page

Reply:
don't know if we still need API docs link in the footer if we have this in the header

Reply:
looks like you only added a link in the footer.

Reply:
I don't think docs deserve to be in the header. It's completely irrelevant for site users.

Reply:
i'm saying add the API key page to the header and remove the docs link (since the key page already contains a link to docs).

Reply:
fixed. SEO for docs is very high, so i want to keep it in the footer.